### PR TITLE
fix: dto types changed

### PR DESCRIPTION
**Pull request by jeremybarbet**
## What

- fix broken build

## Checklist:

- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] Formatting passes locally with my changes
- [ ] I have rebased against master before asking for a review

### CODE REVIEW

File: apps/application-system/api/src/app/modules/application/dto/populateExternalData.dto.ts [hunk 1]
```diff
@@ -2,6 +2,8 @@ import { IsArray, IsString } from 'class-validator'
 import { ApiProperty } from '@nestjs/swagger'
 import { Type } from 'class-transformer'
 
+console.log('-trigger schemas')
```
Comment (arnifreyrs) [inline apps/application-system/api/src/app/modules/application/dto/populateExternalData.dto.ts:5]: remove this?

File: scripts/schemas.js [hunk 2]
```diff
@@ -2,6 +2,8 @@ const { stat, writeFile } = require('fs')
 const { exec } = require('child_process')
 const { promisify } = require('util')
 
+console.log('-trigger')
```
Comment (arnifreyrs) [inline scripts/schemas.js:5]: and this or?

**Review by arnifreyrs — APPROVED** [review]
nice nice!

File: apps/application-system/api/src/app/modules/application/dto/populateExternalData.dto.ts (hunk 1 above)
Comment (jeremybarbet) [reply apps/application-system/api/src/app/modules/application/dto/populateExternalData.dto.ts:5]: Still work in progress, the file is not detected when it is updated and doesn't trigger cache/hash-key zip

**Pull request closed without merging**